src/terra-dev-site/app/components.2/WorkspaceContent.app.mdx

```diff
@@ -0,0 +1,49 @@
+import WorkspaceContentProps from 'terra-application/lib/workspace/WorkspaceContent?dev-site-props-table';
+import WorkspaceContentStatusOverlayProps from 'terra-application/lib/workspace/overlays/WorkspaceContentStatusOverlay?dev-site-props-table';
+import WorkspaceContentStatusOverlayButtonProps from 'terra-application/lib/workspace/shared/StatusLayoutButton?dev-site-props-table';
+import WorkspaceContentActivityOverlayProps from 'terra-application/lib/workspace/overlays/WorkspaceContentActivityOverlay?dev-site-props-table';
+
+import { Notice } from '@cerner/terra-docs';
+
+# WorkspaceContent
+
+The WorkspaceContent component is essential for integrating into the workspace. It provides the necessary UI and API hooks to present content consistently in the workspace ecosystem. 
+
+## Usage
+
+```jsx
+import { WorkspaceContent } from 'terra-application/lib/workspace';
+```
```

Comment:
these usage sections are pretty useless, should we ditch it?

Reply:
maybe it's useful in showing the path to get to the workspace component, but thats about it.

Reply:
I think that's all they ever did, across all our packages. "Usage" might be the wrong word, but showing the the import path is helpful.